internal/provider/data_source_dirhash_sha256.go

```diff
@@ -35,26 +35,24 @@ func dataSourceDirhashSha256() *schema.Resource {
 	}
 }
 
-func dataSourceDirhashSha256Read(ctx context.Context, d *schema.ResourceData, meta any) diag.Diagnostics {
-	var dir = d.Get("directory").(string)
-	d.SetId(dir)
-	var ignore = []string{}
-	if v, ok := d.GetOk("ignore"); ok {
-		ignore = stringListToStringSlice(v.([]interface{}))
+func dataSourceDirhashSha256Read(ctx context.Context, dr *schema.ResourceData, meta any) diag.Diagnostics {
```

Comment:
Hate to be annoying about this... but i strongly suggest it be name `rd` and not `rd`

Reply:
That's my mistake, you're not being annoying, I wrote it wrong

Reply:
That's what reviews are for